### ISSUE TITLE
Consider blocking Luminati network

### ISSUE THREAD

**Issue by durablenapkin** [lightswitch05/hosts#129]
Just wanted to share this with you: https://old.reddit.com/r/oisd_blocklist/comments/dwxgld/dbloisdnl_internets_1_domain_blocklist/fgkm522/

Domains:

```
lumtest.com
proxyjs.lum-sdk.io
proxyjs.luminatinet.com
http-test1.luminatinet.com
perr.lum-sdk.io
perr.l-agent.me
dns-test1.hola.org
clientsdk.luminatinet.com
clientsdk.lum-sdk.io
clientsdk.luminati-china.io
```

A couple more listed here: https://www.threatcrowd.org/ip.php?ip=54.243.128.120

Such as:

```
client.luminati.io
perr.luminati.io
```

And a whole bunch of hola domains (since hola vpn uses the Luminati network).

I'd be curious to see what comes up if you expand: lumtest.com, l-agent.me, lum-sdk.io, luminati.io, luminati-china.io, svd-cdn.com, and luminatinet.com

For your consideration :)

**Comment by lightswitch05** [comment]
Hola: 
> One of the most popular free VPN/geo-unblocking solutions on the planet is operated by Israel-based Hola. It can be added to most popular browsers in seconds and has an impressive seven million users on Chrome alone. Overall the company boasts 46 million users of its service.

Also:

> “Hola has gotten greedy. They recently (late 2014) realized that they basically have a 9 million IP strong botnet on their hands, and they began selling access to this botnet (right now, for HTTP requests only) at luminati [dot] io,” the 8chan owner says.

Source: https://torrentfreak.com/hola-vpn-sells-users-bandwidth-150528/

While I agree that these services are terrible and I would not want them to be able to run on my network - I don't believe I currently have an appropriate place to add them. They are not advertising, not tracking, not AMP, and not facebook. So I won't be adding them, but if I ever get a list that these make sense in, I'll be sure to come back and add them. Thank you for the report, it was interesting to learn about these two services.

Since you are interested, I ran the hosts through and these are the ones it found. The tool is limited to 100 subdomains per run (virustotal API) - so there are a whole lot more subdomains out there then these, but these are the ones it found.

```
luminatinet.com
104-131-216-12.luminatinet.com
104-131-221-237.luminatinet.com
104-248-0-6.luminatinet.com
104-248-129-135.luminatinet.com
134-209-167-144.luminatinet.com
134-209-167-220.luminatinet.com
134-209-190-209.luminatinet.com
134-209-211-112.luminatinet.com
134-209-224-234.luminatinet.com
134-209-244-34.luminatinet.com
138-68-211-82.luminatinet.com
138-68-215-160.luminatinet.com
142-93-176-7.luminatinet.com
142-93-245-44.luminatinet.com
142-93-56-172.luminatinet.com
157-230-170-87.luminatinet.com
157-230-244-216.luminatinet.com
157-245-73-86.luminatinet.com
157-245-77-17.luminatinet.com
159-65-182-99.luminatinet.com
159-65-185-116.luminatinet.com
159-89-112-125.luminatinet.com
159-89-120-51.luminatinet.com
159-89-55-252.luminatinet.com
159-89-88-66.luminatinet.com
159-89-88-68.luminatinet.com
159-89-90-118.luminatinet.com
162-243-12-62.luminatinet.com
162-243-13-22.luminatinet.com
165-22-105-233.luminatinet.com
165-22-113-57.luminatinet.com
165-22-12-205.luminatinet.com
165-22-12-84.luminatinet.com
165-22-122-26.luminatinet.com
165-22-122-50.luminatinet.com
165-22-139-31.luminatinet.com
165-22-156-28.luminatinet.com
165-22-159-165.luminatinet.com
165-22-160-178.luminatinet.com
165-22-175-52.luminatinet.com
165-22-182-26.luminatinet.com
165-22-190-253.luminatinet.com
165-22-23-51.luminatinet.com
165-22-3-62.luminatinet.com
165-22-36-148.luminatinet.com
165-22-47-231.luminatinet.com
165-227-182-85.luminatinet.com
165-227-196-187.luminatinet.com
165-227-38-221.luminatinet.com
165-227-46-10.luminatinet.com
165-227-90-252.luminatinet.com
167-172-176-11.luminatinet.com
167-71-105-77.luminatinet.com
167-71-129-34.luminatinet.com
167-71-170-149.luminatinet.com
167-71-186-47.luminatinet.com
167-71-205-98.luminatinet.com
167-71-241-8.luminatinet.com
167-71-244-41.luminatinet.com
167-71-254-15.luminatinet.com
167-71-4-214.luminatinet.com
167-71-45-81.luminatinet.com
167-71-56-61.luminatinet.com
167-71-85-122.luminatinet.com
167-71-87-68.luminatinet.com
167-99-131-29.luminatinet.com
167-99-183-170.luminatinet.com
167-99-195-224.luminatinet.com
167-99-195-229.luminatinet.com
178-128-163-121.luminatinet.com
178-128-172-236.luminatinet.com
178-128-66-223.luminatinet.com
178-62-180-117.luminatinet.com
178-62-181-30.luminatinet.com
178-62-181-37.luminatinet.com
178-62-184-167.luminatinet.com
178-62-187-189.luminatinet.com
188-226-221-142.luminatinet.com
192-241-188-159.luminatinet.com
198-199-105-122.luminatinet.com
198-199-124-43.luminatinet.com
198-199-93-251.luminatinet.com
206-189-106-182.luminatinet.com
206-189-150-238.luminatinet.com
209-97-178-54.luminatinet.com
37-139-5-232.luminatinet.com
67-207-81-234.luminatinet.com
67-207-88-227.luminatinet.com
68-183-0-241.luminatinet.com
68-183-147-192.luminatinet.com
68-183-159-111.luminatinet.com
68-183-199-153.luminatinet.com
68-183-200-36.luminatinet.com
68-183-41-227.luminatinet.com
68-183-65-97.luminatinet.com
client.luminatinet.com
clientsdk.luminatinet.com
http-test1.luminatinet.com
perr.luminatinet.com
proxyjs.luminatinet.com
lumtest.com
www.lumtest.com
lum-sdk.io
104-131-216-207.lum-sdk.io
104-131-220-236.lum-sdk.io
104-131-221-175.lum-sdk.io
104-131-221-187.lum-sdk.io
104-131-222-40.lum-sdk.io
104-131-223-61.lum-sdk.io
104-248-157-141.lum-sdk.io
104-248-58-24.lum-sdk.io
134-209-102-76.lum-sdk.io
134-209-188-17.lum-sdk.io
134-209-211-112.lum-sdk.io
134-209-224-131.lum-sdk.io
134-209-243-214.lum-sdk.io
134-209-25-52.lum-sdk.io
134-209-26-233.lum-sdk.io
134-209-61-138.lum-sdk.io
134-209-97-112.lum-sdk.io
138-68-219-183.lum-sdk.io
138-68-222-241.lum-sdk.io
138-68-222-251.lum-sdk.io
142-93-84-17.lum-sdk.io
142-93-86-14.lum-sdk.io
157-230-104-246.lum-sdk.io
157-230-208-240.lum-sdk.io
157-230-221-63.lum-sdk.io
157-230-233-96.lum-sdk.io
157-230-239-144.lum-sdk.io
157-230-251-42.lum-sdk.io
157-245-102-140.lum-sdk.io
157-245-104-24.lum-sdk.io
157-245-67-104.lum-sdk.io
157-245-69-169.lum-sdk.io
157-245-70-97.lum-sdk.io
159-89-85-187.lum-sdk.io
162-243-12-91.lum-sdk.io
162-243-13-57.lum-sdk.io
162-243-136-111.lum-sdk.io
162-243-137-175.lum-sdk.io
162-243-137-70.lum-sdk.io
162-243-27-136.lum-sdk.io
165-22-116-149.lum-sdk.io
165-22-116-190.lum-sdk.io
165-22-116-79.lum-sdk.io
165-22-121-43.lum-sdk.io
165-22-145-124.lum-sdk.io
165-22-149-216.lum-sdk.io
165-22-156-28.lum-sdk.io
165-22-180-131.lum-sdk.io
165-22-182-135.lum-sdk.io
165-22-36-27.lum-sdk.io
165-22-49-28.lum-sdk.io
165-22-85-142.lum-sdk.io
165-227-114-232.lum-sdk.io
165-227-199-225.lum-sdk.io
165-227-207-138.lum-sdk.io
165-227-41-179.lum-sdk.io
165-227-41-187.lum-sdk.io
165-227-90-105.lum-sdk.io
165-227-90-138.lum-sdk.io
165-227-90-81.lum-sdk.io
167-172-252-218.lum-sdk.io
167-71-108-33.lum-sdk.io
167-71-111-75.lum-sdk.io
167-71-129-34.lum-sdk.io
167-71-136-121.lum-sdk.io
167-71-249-247.lum-sdk.io
167-71-81-40.lum-sdk.io
167-99-127-58.lum-sdk.io
167-99-131-46.lum-sdk.io
167-99-131-72.lum-sdk.io
167-99-72-224.lum-sdk.io
178-128-30-212.lum-sdk.io
178-62-180-237.lum-sdk.io
178-62-180-243.lum-sdk.io
178-62-188-105.lum-sdk.io
178-62-189-27.lum-sdk.io
192-241-161-143.lum-sdk.io
192-241-188-159.lum-sdk.io
192-241-241-61.lum-sdk.io
192-241-246-169.lum-sdk.io
206-189-107-93.lum-sdk.io
206-189-110-93.lum-sdk.io
206-189-17-114.lum-sdk.io
206-81-18-82.lum-sdk.io
206-81-24-146.lum-sdk.io
206-81-26-161.lum-sdk.io
37-139-6-186.lum-sdk.io
37-139-6-76.lum-sdk.io
68-183-0-241.lum-sdk.io
68-183-153-117.lum-sdk.io
68-183-195-37.lum-sdk.io
68-183-195-99.lum-sdk.io
68-183-196-3.lum-sdk.io
68-183-199-153.lum-sdk.io
68-183-36-254.lum-sdk.io
68-183-70-176.lum-sdk.io
82-196-6-205.lum-sdk.io
clientsdk.lum-sdk.io
perr.lum-sdk.io
proxyjs.lum-sdk.io
www.lum-sdk.io
luminati.io
campaign.luminati.io
cdn.luminati.io
client.luminati.io
clientsdk.luminati.io
perr.luminati.io
proxyjs.luminati.io
start.luminati.io
url7379.luminati.io
web.luminati.io
www.luminati.io
zproxy.luminati.io
servercountry-au.zproxy.luminati.io
servercountry-gb.zproxy.luminati.io
servercountry-us.zproxy.luminati.io
servercountry-us-session-101156.zproxy.luminati.io
servercountry-us-session-102659.zproxy.luminati.io
servercountry-us-session-109226.zproxy.luminati.io
servercountry-us-session-110072.zproxy.luminati.io
servercountry-us-session-110722.zproxy.luminati.io
servercountry-us-session-110772.zproxy.luminati.io
servercountry-us-session-111460.zproxy.luminati.io
servercountry-us-session-113461.zproxy.luminati.io
servercountry-us-session-123134.zproxy.luminati.io
servercountry-us-session-124052.zproxy.luminati.io
servercountry-us-session-124263.zproxy.luminati.io
servercountry-us-session-126363.zproxy.luminati.io
servercountry-us-session-12639.zproxy.luminati.io
servercountry-us-session-127922.zproxy.luminati.io
servercountry-us-session-130479.zproxy.luminati.io
servercountry-us-session-130588.zproxy.luminati.io
servercountry-us-session-13239.zproxy.luminati.io
servercountry-us-session-135644.zproxy.luminati.io
servercountry-us-session-136115.zproxy.luminati.io
servercountry-us-session-137948.zproxy.luminati.io
servercountry-us-session-139229.zproxy.luminati.io
servercountry-us-session-140781.zproxy.luminati.io
servercountry-us-session-142474.zproxy.luminati.io
servercountry-us-session-146289.zproxy.luminati.io
servercountry-us-session-150800.zproxy.luminati.io
servercountry-us-session-155457.zproxy.luminati.io
servercountry-us-session-15574.zproxy.luminati.io
servercountry-us-session-160620.zproxy.luminati.io
servercountry-us-session-161635.zproxy.luminati.io
servercountry-us-session-163823.zproxy.luminati.io
servercountry-us-session-163956.zproxy.luminati.io
servercountry-us-session-167468.zproxy.luminati.io
servercountry-us-session-168716.zproxy.luminati.io
servercountry-us-session-169570.zproxy.luminati.io
servercountry-us-session-169954.zproxy.luminati.io
servercountry-us-session-172636.zproxy.luminati.io
servercountry-us-session-173351.zproxy.luminati.io
servercountry-us-session-174984.zproxy.luminati.io
servercountry-us-session-175273.zproxy.luminati.io
servercountry-us-session-17777.zproxy.luminati.io
servercountry-us-session-179342.zproxy.luminati.io
servercountry-us-session-187172.zproxy.luminati.io
servercountry-us-session-187985.zproxy.luminati.io
servercountry-us-session-18802.zproxy.luminati.io
servercountry-us-session-21269.zproxy.luminati.io
servercountry-us-session-22856.zproxy.luminati.io
servercountry-us-session-23918.zproxy.luminati.io
servercountry-us-session-24389.zproxy.luminati.io
servercountry-us-session-27014.zproxy.luminati.io
servercountry-us-session-28845.zproxy.luminati.io
servercountry-us-session-29854.zproxy.luminati.io
servercountry-us-session-30180.zproxy.luminati.io
servercountry-us-session-36412.zproxy.luminati.io
servercountry-us-session-51385.zproxy.luminati.io
servercountry-us-session-53910.zproxy.luminati.io
servercountry-us-session-57407.zproxy.luminati.io
servercountry-us-session-61927.zproxy.luminati.io
servercountry-us-session-63819.zproxy.luminati.io
servercountry-us-session-68103.zproxy.luminati.io
servercountry-us-session-69070.zproxy.luminati.io
servercountry-us-session-71938.zproxy.luminati.io
servercountry-us-session-72731.zproxy.luminati.io
servercountry-us-session-73017.zproxy.luminati.io
servercountry-us-session-73347.zproxy.luminati.io
servercountry-us-session-74936.zproxy.luminati.io
servercountry-us-session-78392.zproxy.luminati.io
servercountry-us-session-82482.zproxy.luminati.io
servercountry-us-session-830.zproxy.luminati.io
servercountry-us-session-8436.zproxy.luminati.io
servercountry-us-session-86024.zproxy.luminati.io
servercountry-us-session-87047.zproxy.luminati.io
servercountry-us-session-93272.zproxy.luminati.io
servercountry-us-session-93484.zproxy.luminati.io
servercountry-us-session-95753.zproxy.luminati.io
servercountry-us-session-96985.zproxy.luminati.io
servercountry-us-session-972.zproxy.luminati.io
servercountry-us-session-9734.zproxy.luminati.io
servercountry-us-session-97563.zproxy.luminati.io
servercountry-us-session-99150.zproxy.luminati.io
servercountry-us-session-99236.zproxy.luminati.io
session-631875.zproxy.luminati.io
session-633762.zproxy.luminati.io
session-702990377.zproxy.luminati.io
session-790830335.zproxy.luminati.io
luminati-china.io
cdn.luminati-china.io
clientsdk.luminati-china.io
perr.luminati-china.io
www.luminati-china.io
zproxy.luminati-china.io
perr.l-agent.me
hola.org
cdn4.hola.org
client.hola.org
client-cdn4.hola.org
dns-test1.hola.org
perr.hola.org
web.hola.org
www.hola.org
zagent1093.hola.org
zagent1126.hola.org
zagent1154.hola.org
zagent1172.hola.org
zagent1266.hola.org
zagent140.hola.org
zagent144.hola.org
zagent145.hola.org
zagent146.hola.org
zagent149.hola.org
zagent1602.hola.org
zagent1612.hola.org
zagent1615.hola.org
zagent1616.hola.org
zagent1617.hola.org
zagent1621.hola.org
zagent1623.hola.org
zagent1626.hola.org
zagent1656.hola.org
zagent1666.hola.org
zagent1706.hola.org
zagent1707.hola.org
zagent173.hola.org
zagent174.hola.org
zagent175.hola.org
zagent1766.hola.org
zagent1767.hola.org
zagent1769.hola.org
zagent185.hola.org
zagent1870.hola.org
zagent1873.hola.org
zagent1920.hola.org
zagent196.hola.org
zagent1962.hola.org
zagent1965.hola.org
zagent197.hola.org
zagent1989.hola.org
zagent1993.hola.org
zagent1995.hola.org
zagent1997.hola.org
zagent1999.hola.org
zagent200.hola.org
zagent2006.hola.org
zagent2021.hola.org
zagent2022.hola.org
zagent2026.hola.org
zagent2028.hola.org
zagent2030.hola.org
zagent2036.hola.org
zagent2039.hola.org
zagent2041.hola.org
zagent2042.hola.org
zagent2046.hola.org
zagent2049.hola.org
zagent205.hola.org
zagent2050.hola.org
zagent2051.hola.org
zagent2053.hola.org
zagent2054.hola.org
zagent2063.hola.org
zagent2066.hola.org
zagent2070.hola.org
zagent211.hola.org
zagent254.hola.org
zagent279.hola.org
zagent29.hola.org
zagent444.hola.org
zagent45.hola.org
zagent464.hola.org
zagent513.hola.org
zagent518.hola.org
zagent538.hola.org
zagent540.hola.org
zagent546.hola.org
zagent547.hola.org
zagent554.hola.org
zagent63.hola.org
zagent667.hola.org
zagent697.hola.org
zagent750.hola.org
zagent753.hola.org
zagent763.hola.org
zagent767.hola.org
zagent770.hola.org
zagent772.hola.org
zagent779.hola.org
zagent789.hola.org
zagent925.hola.org
zagent926.hola.org
zagent96.hola.org
zagent984.hola.org
zagent987.hola.org
zs-wbm-hola.hola.org
zserver.hola.org
```

**Comment by durablenapkin** [comment]
Thanks for your time looking into this! I agree that this is out of scope for your current lists.

Over the weekend I used your Python script to expand domains and created this list which I am using as a source in my personal blocklist for my parents (I'd hate for them to become a part of a botnet): https://github.com/durablenapkin/block/blob/master/luminati.txt

Thanks again!
